GameData/RealismOverhaul/RO_SuggestedMods/Hephaistos/RO_CentaurV.cfg

```diff
@@ -0,0 +1,293 @@
+//	================================================================================
+//	Centaur V
+//	================================================================================
+
+
+// Centaur V LH2 Tank
+@PART[SP_largeACESTank]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@rescaleFactor = 1.742
+	@mass = 1.956
+	@title = Centaur V LH2 Tank
+	@manufacturer = United Launch Alliance
+	@description = One massive and extremely cold tank of Liquid Hydrogen, make sure you put insulation on it before it all leaks away!
+	%skinTempTag = Steel
+	%internalTempTag = Steel
+	!RESOURCE[*],* {}
+	MODULE
+	{
+		name = ModuleFuelTanks
+		type = Cryogenic
+		volume = 104000
+		basemass = -1
+	}
+}
+
+// Centaur V LOX Tank
+@PART[SP_smallACESTank]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@rescaleFactor = 1.742
+	@mass = 0.978
+	@title = Centaur V LOX Tank
+	@manufacturer = United Launch Alliance
+	@description = A slightly smaller but still extremely cold tank, this time filled with Liquid Oxygen, might still want some insulation on this thing!
+	%skinTempTag = Steel
+	%internalTempTag = Steel
+	!RESOURCE[*],* {}
+	MODULE
+	{
+		name = ModuleFuelTanks
+		type = Cryogenic
+		volume = 52000
+		basemass = -1
+	}
+}
+
+// Centaur V Aft Bulkhead
+@PART[SP_vulcanQuadAdaptor]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@rescaleFactor = 1.742
+	@mass = 0.535
+	@title = Centaur V Engine Mount
+	@manufacturer = United Launch Alliance
+	@description = A mount to attach your RL10C-1-1 or RL10C-X engines, comes with RCS to orient your vehicle.
+	%skinTempTag = Steel
+	%internalTempTag = Instruments
+	!RESOURCE[Monopropellant] {}
+	MODULE
+	{
+		name = ModuleFuelTanks
+		type = Tank-Sep-AlCu-HP
+		volume = 300
+		basemass = -1
+		TANK
+		{
+			name = Hydrogen
+			amount = 44199
+			maxAmount = 44199
+		}
+		TANK
+		{
+			name = Oxygen
+			amount = 15780
+			maxAmount = 15780
+		}
+	}
+	@MODULE[ModuleRCSFX]
+	{
+		@thrusterPower = 0.2
+		!resourceName = DELETE
+		@resourceFlowMode = STACK_PRIORITY_SEARCH
+		PROPELLANT
+		{
+			name = Hydrogen
+			ratio = 0.7369
+		}
+		PROPELLANT
+		{
+			name = Oxygen
+			ratio = 0.2631
+		}
+	}
+	RESOURCE
+	{
+		name = ElectricCharge
+		amount = 25000
+		maxAmount = 25000
+	}
+	!MODULE[ModuleCommand] {}
+	MODULE
+	{
+		name = ModuleCommand
+		minimumCrew = 0
+		RESOURCE
+		{
+			name = ElectricCharge
+			rate = 0.020
+		}
+		hasHibernation = True
+	}
+	MODULE
+	{
+		name = ModuleRealAntenna
+		antennaDiameter = 1.5
+		RFBand = UHF
+	}
+}
+
+// Vulcan Fairing
+@PART[SP_vulcanPayloadFairing]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@rescaleFactor = 1.742
+	@mass = 0.25
+	@title = Vulcan Centaur 5.4m PLF
+	@manufacturer = United Launch Alliance
+	@description = One Large Fairing to store all your valuble satellites
+	%ROSAFRescale = 1.742
+	@attachRules = 1,1,1,1,0
+	%skinTempTag = HRSI

```

Comment:
Is this needed?

Reply:
I assume you mean the HRSI tag, in which case yes.

The Attach Rules edit is to allow radial placement of decals